README.md

```diff
@@ -11,14 +11,14 @@ _Take control of your internal daemons!_
 
 Pebble's key features:
 
-- [Layer](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layers/)-based configuration
+- [Layer](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layer-specification/)-based configuration
 - Service [dependencies](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/service-dependencies/)
-- Service [logs](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/logs/) and [log forwarding](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/log-forwarding/)
+- Service [logs](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/#logs) and [log forwarding](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/log-forwarding/)
 - [Health checks](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/health-checks/)
 - [Notices](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/notices/) (aggregated events)
 - [Identities](https://canonical-pebble.readthedocs-hosted.com/en/latest/how-to/manage-identities/)
 - Can be used in virtual machines and [containers](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/pebble-in-containers/)
```

Comment:
This link (https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/pebble-in-containers/) is also broken, but I couldn't find the page that this should point to.